src/index.js

```diff
@@ -216,7 +227,16 @@ function shimBelongsTo(target) {
           return Promise.resolve(null);
         }
 
-        const loader = options[EXPECTED_OPTIONS_KEY].loaders[this.target.name].bySingleAttribute[this.targetKey];
+        const loaders = options[EXPECTED_OPTIONS_KEY].loaders;
+        let loader = loaders[this.target.name].bySingleAttribute[this.targetKey];
+        if (options.paranoid === false) {
```

Comment:
We should probably follow the same pattern and consider `options.raw` a special case as well.

Reply:
I followed the same pattern of the solution on `shimModel`. The `options.raw` was already there, the person who solved the paranoid issue only added `|| options.paranoid === false`, as I did in this branch.

Reply:
Correct, but I think that person missed a few things. `shimModel` has `options.raw` as a special case so queries cached with `raw` attribute are not mixed with queries without the raw attribute. Same for `paranoid`.

It seems that when the person tried to fix the paranoid issue, they didn't extend the `raw` protection to other shims and also didn't fix the paranoid problem for all types of association.

Reply:
I think you may have overlooked my comment here:

> I think the bug still persists for other types of relationships. It seems that shimHasOne, shimHasMany and shimBelongsToMany are not taking paranoid into account.

Reply:
>Correct, but I think that person missed a few things. shimModel has options.raw as a special case so queries cached with >raw attribute are not mixed with queries without the raw attribute. Same for paranoid.
>It seems that when the person tried to fix the paranoid issue, they didn't extend the raw protection to other shims and also >didn't fix the paranoid problem for all types of association.

Ok, I'll use raw too then.

Reply:
>I think you may have overlooked my comment here:

I didn't, I'm working on it since last night.